src/libs/Navigation/AppNavigator/createResponsiveStackNavigator/ThreePaneView.js

```diff
@@ -66,12 +66,24 @@ function ThreePaneView(props) {
                                     StyleUtils.displayIfTrue(props.state.index === i),
                                 ]}
                             >
-                                <PressableWithoutFeedback
-                                    style={[styles.flex1]}
-                                    onPress={() => props.navigation.goBack()}
-                                    accessibilityLabel={translate('common.close')}
-                                    accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
-                                />
+                                <View style={[styles.flex1, styles.flexColumn]}>
+                                    {/* The PressableWithoutFeedback with 30px height was added to make whole dimmed area pressable
+                                    on all web devices, but aslo make the top of the desktop apps draggable.
+                                    Please remember that in latestest Electron update, buttons can't be both draggable and clickable  */}
+                                    <PressableWithoutFeedback
```

Comment:
So just to be clear, does this mean that this `PressableWithoutFeedback` w/ static 30px height is draggable but not clickable?

Reply:
(btw if we go with this approach we need to move the style to be in `styles.js` instead of inline)

Reply:
@roryabraham yes but only on the desktop app. I put two Pressables to make everything behave like it is one big Pressable (like before it was before) on other devices. But on the desktop app it will have 30px of draggable, not clickable bar at the top